README.md

```diff
@@ -6,6 +6,23 @@
 [![Translation](https://hosted.weblate.org/widgets/kaidan/-/svg-badge.svg)](https://hosted.weblate.org/projects/kaidan/translations/)
 [![License](https://img.shields.io/badge/license-GPLv3+ / CC BY‐SA 4.0-blue.svg)](https://raw.githubusercontent.com/kaidanim/kaidan/master/LICENSE.txt)
 
+
+## About
+
+Kaidan is a simple, user-friendly Jabber/XMPP client providing a modern user-interface
+using [Kirigami](https://techbase.kde.org/Kirigami) and [QtQuick](http://wiki.qt.io/Qt_Quick).
+The back-end of Kaidan is completely written in C++ using the [Swiften](http://swift.im/swiften.html)
+Library of the [Swift Instant Messenger](http://swift.im/swift.html) and [Qt 5](https://www.qt.io/).
+
+Currently Kaidan only has been tested on GNU/Linux Desktops, but because of the use of Qt we shouldn't have
+any problems porting it to all other common platforms, even Android. Nevertheless we will work on another
+GUI using Silica for better look & feel on Sailfish OS soon (See [KaidanSF](https://github.com/KaidanIM/KaidanSF)).
+But if you want to get a client for Sailfish OS _now_, maybe take a look at
+[Shmoose](https://github.com/geobra/harbour-shmoose), a fork by an earlier developer of Kaidan.
+
+Kaidan is NOT finished at all and we still have much to do!
+
+
 ## Installation
```

Comment:
You don't have to know how to build Kaidan packages, if we've already done that (and published them to https://github.com/KaidanIM/packages)